### PR TITLE
develop/feature/base-method-abstraction

### PR DESCRIPTION
**Pull request by unrealwork**
Created class BaseMethod for non api requests

### CODE REVIEW

File: src/test/java/com/axibase/tsd/base/BaseMethod.java [hunk 1]
```diff
@@ -21,26 +20,33 @@
...
 
     static {
         java.util.logging.LogManager.getLogManager().reset();
         SLF4JBridgeHandler.removeHandlersForRootLogger();
         SLF4JBridgeHandler.install();
         java.util.logging.Logger.getLogger("").setLevel(Level.FINEST);
-
+        try {
+            prepare();
+        } catch (FileNotFoundException e) {
```
Comment (KutuzovGregory) [inline src/test/java/com/axibase/tsd/base/BaseMethod.java]: there is no reason to catch this exception

**Comment by KutuzovGregory** [comment]
Ok

**Comment by rmakulov** [comment]
Ok.

**Comment by unrealwork** [comment]
Merged with master without any conflicts

**Comment by Sild** [comment]
Maybe we should change package tree like the following:

```
builder.\.\.
model.\.\.
transport.tcp
action.api
action.root
```
?

**Pull request merged**